lld/ELF/InputFiles.cpp

```diff
@@ -775,6 +807,28 @@ void ObjFile<ELFT>::initializeSections(bool ignoreComdats,
       continue;
     }
 
+    // Extract Build Attributes section contents into aarch64BAsubSections.
+    // Input objects may contain both build Build Attributes and GNU
+    // properties. We delay processing Build Attributes until we have finished
+    // reading all sections so that we can check that these are consistent.
+    //
+    // Ideally this belongs in the following switch, but due to the
```

Comment:
I'd remove "Ideally this belongs in the following switch, but due to the" and change the comment order:

```
// Processor-specific types that do not use the following switch statement.

// Extract Build Attributes section contents. Input objects may contain both build Build Attributes and GNU ...
```